modules/tracker/views/producer/aggregatemetric.phtml

```diff
@@ -148,4 +135,48 @@ echo '<script type="text/javascript" src="'.$this->moduleWebroot.'/public/js/pro
             <input id="aggregateMetricSpecCancel" type="button" value="Cancel" />
         </div>
     </div>
+    <div id='aggregateMetricUserAlerts' style='display: none;'>
+        <div><span>Alerts for aggregate metric spec <label id='aggregateMetricUserAlertsSpecName'></label></span><div>
+        <div><input id='aggregateMetricUserAlertsSpec' type="text" disabled="disabled"/></div>
+        <div id='aggregateMetricUserAlertsThreshold'>
+            Alert Threshold
+            <table id='aggregateMetricUserAlertsThresholdDefinition'>
+                <tbody>
+                    <tr>
+                        <td>comparison</td>
+                        <td><input id='aggregateMetricSpecAlertComparison' type="text" disabled="disabled"/></td>
+                    </tr>
+                    <tr>
+                        <td>value</td>
+                        <td><input id='aggregateMetricSpecAlertValue' type="text" disabled="disabled"/></td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+        <table id='aggregateMetricSpecAlertedUsers'>
+            <thead>
+                <tr>
+                    <th>Alerted Users</th>
+                    <th>Actions</th>
+                </tr>
+                <tbody>
+                </tbody>
+        </table>
+        <div id="addAggregateMetricSpecAlertUser">
+            Add a user to be alerted<br/>
+            <div class='alertUserSearch'>
+                    <input type='text' id='addAlertUserSearch' value='' autocomplete='off'
+                       autocorrect='off' autocapitalize='off'/>
```

Comment:
I associate these as fields added to `input`s that make things behave better on mobile browsers. That's all.